bazel/glog.bzl

```diff
@@ -128,7 +128,7 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
             "src/stacktrace_powerpc-inl.h",
             "src/stacktrace_windows-inl.h",
             "src/stacktrace_x86-inl.h",
-            "src/stacktrace_x86_64-inl.h",
```

Comment:
Why did you remove the header from the list?

Reply:
I renamed it to `stacktrace_unwind-inl.h` as it actually uses `_Unwind_*` API rather than hand-coded like `stacktrace_x86-inl.h`.